scripts/gen_ta_src.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/python3
+'''
+Script gen_ta_src.py used for generating source files which creates lookup table
+of loadable ta's. This table used by tee supplicant to load ta to OP-TEE
+Options:
+  -u [UUIDS ...], --uuids [UUIDS ...]
+                        List of enabled layers
+  -f FILE, --file FILE  Config file with enabled layers
+  -o OUT, --out OUT     Output file name (without extension)
+  -s SOURCE, --source SOURCE
+                        Directory with prebuilt ta
+  -t TABLE, --table TABLE
+                        table variable name
+List ot ta can be passed to script follows ways:
+  - with option -u can be pointed in command line
+  - with option -f list of uuids can be enumerated in text file
+  - with option -s script reads all files with extension .ta and used
+                   it as list of enabled uuids
+
+Script generates two files with extensions .c and .S and name pointed with -o option
+To use lookup table in application genarated files just should be build with application.
```

Comment:
There should be more comprehensible usage description. Also there should be natural way to find this information for the potential user. First of all user should know which script is they looking for. This information can be get from README. Then call <scriptname> --help or <scriptname> -h to get usage information.
Also, this [commit](https://github.com/aoscloud/aos_core_zephyr/pull/83/commits/bc8ee99c66f80cbb8418fccd0edff225d00b131e) provides additional step in CMake to build TA from env var TA_DEPLOY_DIR. This possibility also should be mentioned in readme.
zephyr-optee-test is the potential user of this approach so those changes should be transfered there as well.

Reply:
I don't think this script for public usage. This script used ONLY by build system. If user wants to use this script he has --help option and comment at the beginning of script. Can you find User Manual for scripts in zephyr distribution? Specially for script used by build system?